package.json

```diff
@@ -75,7 +75,8 @@
     "nx": "16.0.1",
     "prettier": "^2.8.4",
     "typescript": "^4.9.5",
-    "vercel": "^30.2.1"
+    "vercel": "^32.1.0",
+    "vite": "^4.4.9"
```

Comment:
What is the reason for having `vite` on root?

Reply:
it is a weird combination... for some reason, upon upgrading `vitest` to a newer version, yarn can't locate the `vite` binary anymore. If I recall correctly, you once experienced this issue.
Putting `vite` as a top-level dependency eliminates this issue.

Reply:
Yeah, I had experienced it back then. I thought this was added to the root mistakenly. If it makes sense, its fine.